### ISSUE TITLE
Express Checkout - Smart Buttons - No Padding/Margin..??

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#1055]
![image](https://user-images.githubusercontent.com/629241/37997137-aab75a02-31df-11e8-8013-f0b2d49991ba.png)

**Comment by angelleye** [comment]
I fixed this by adding this to my theme...

```
.paypal_box_button {
    margin-top:60px;
}
```

I don't think I should have to do that, though.  Can't we get that done from the plugin itself so people don't have to adjust the CSS on their theme for proper spacing?

**Comment by kcppdevelopers** [comment]
@angelleye  I am little bit confused here, We have not used ".paypal_box_button" CSS class in our plugin as well I not found on angelleye.com so could you please let me know where exactly you find that class?

Product Page:

![product](https://user-images.githubusercontent.com/13145461/38011902-e4f7e914-327c-11e8-9901-54f127176ed1.png)

Cart Page:
![cart](https://user-images.githubusercontent.com/13145461/38011910-ee72380a-327c-11e8-842e-87582acf0bcc.png)

Checkout Page:
![checkout](https://user-images.githubusercontent.com/13145461/38011915-f894de78-327c-11e8-9286-9e4ef26ba296.png)

For me it's working fine without any CSS.

**Comment by angelleye** [comment]
@kcppdevelopers I can explain more.

In #1030 you pointed out that I had some custom CSS that was causing the issue I mentioned there.  While reviewing my style.css I noticed I had some other stuff in there that was no longer necessary, so I removed them.  One of those things was this:

```
.paypal_box_button {
    margin-top:60px;
}
```

After I removed that, though, it wound up looking like my screenshot above, where the PayPal buttons are smashed up against the Add to Cart button.  I, too, was confused because I wasn't seeing that class anywhere.  However, that was the only thing I had removed that looked related, so I added it back, and now there is space between the buttons again.
